sdks/python/tox.ini

```diff
@@ -34,6 +34,26 @@ commands =
   python setup.py test
 passenv = TRAVIS*
 
+[testenv:py27cython]
```

Comment:
Please add `whitelist_externals=find` here, otherwise there will be a warning in the output.